src/cli/dev.js

```diff
@@ -7,16 +7,19 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-// Using these modules when in dev mode to extend the stack traces.
-// Why?
-//      When running unattended (non-awaited) promises, V8 removes the parent call-stack (as none of those objects are required by V8).
-//      This results in (very) incomplete and unhelpful stack traces that makes it super-hard to figure out what called a failing promise.
-//      The libraries below help to provide more complete stack traces.
-//
-// `trace` uses AsyncHooks to store the call-stack and attaches them to the stack trace when an error occurs.
-require('trace');
-// `clarify` tries to remove Node.js' internal libraries from the stack trace to make it a bit more human-friendly.
-require('clarify');
+const isDev = process.argv.includes('--dev');
```

Comment:
@afharo I've taken the liberty of only enabling these libs when `--dev` is explicitly set. They're pretty costly in terms of CPU and create a memory leak. If you have a better way to do this, I'm all ears.

Reply:
@afharo you good with this? I'm not sure if this is the right way to do this